__tests__/app.test.js

```diff
@@ -225,3 +225,79 @@ describe('POST: /api/reviews/:review_id/comments', () => {
       });
   });
 });
+
+describe('PATCH /api/reviews/:review_id', () => {
+  it('200: responds with the updated review', () => {
+    const requestBody = {
+      inc_votes: 10,
+    };
+    return request(app)
+      .patch('/api/reviews/1')
+      .send(requestBody)
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review).toBeInstanceOf(Object);
+        expect(review).toMatchObject({
+          review_id: 1,
+          title: 'Agricola',
+          category: 'euro game',
+          designer: 'Uwe Rosenberg',
+          owner: 'mallionaire',
+          review_body: 'Farmyard fun!',
+          review_img_url:
+            'https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700',
+          created_at: '2021-01-18T10:00:20.514Z',
+          votes: 11,
+        });
+      });
+  });
+  it('400: malformed request - request body is missing not null values', () => {
+    const requestBody = {
+      inc_votes: null,
+    };
+    return request(app)
+      .patch('/api/reviews/1')
+      .send(requestBody)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Malformed Request');
+      });
+  });
+  it('400: invalid review id', () => {
+    const requestBody = {
+      inc_votes: 10,
+    };
+    return request(app)
+      .patch('/api/reviews/banana')
+      .send(requestBody)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad Request');
+      });
+  });
+  it('400: incorrect datatype in req body', () => {
+    const requestBody = {
+      inc_votes: 'word',
+    };
+    return request(app)
+      .patch('/api/reviews/1')
+      .send(requestBody)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad Request');
+      });
+  });
+  it('404: valid id but not in the db', () => {
+    const requestBody = {
+      inc_votes: 10,
+    };
```

Comment:
I would change this to -Status 404, non-existent ID, e.g. 0 or 9999 to make it clearer.